Meme/Meme/ViewController.swift

```diff
@@ -10,10 +10,49 @@ import UIKit
 
 class ViewController: UIViewController {
 
+    let viewModel: ViewModel
+    var memes = [Meme]()
```

Comment:
We can wrap these in observables and get you on your way to using MVVM with reactive bindings.

Reply:
:Mindblown: